### ISSUE TITLE
Close button in overlay dialogs doesn't have automation properties

### ISSUE THREAD

**Issue by jaigak** [BreeceW/Oxidized#99]
**Describe the bug**
Since the close button is icon only, you should set AutomationProperties.Name to the name of the button. Today it's not set so screen reads will simply say 'close'.

**To Reproduce**
1. Launch Narrator or any other screen reader.
2. Open an overlay dialog like the settings dialog.
3. Press tab to reach the close button.
4. Notice that the screen reader simply says button instead of 'Close button'.

**System(s) that problem occurred on**
Windows 11 Insider preview build 22000.61, PC x64

**Comment by BreeceW** [comment]
Thanks for pointing this out. There are a lot of controls that do not yet have the proper automation properties.
